### ISSUE TITLE
Feed Me integration not accounting for extensions on Vizy field

### ISSUE THREAD

**Issue by robmcfadden** [verbb/vizy#289]
### Describe the bug

We have added an extension to Vizy to allow for id attributes on HTML elements per the [comment here](https://github.com/verbb/vizy/issues/255): "The way around this - while a little more work - is to create an extension for Tiptap that adds this allowed attribute to the Paragraph node."

This works for us when saving content manually in Craft, but not when importing content with id attributes on HTML elements through Feed Me. I believe the [Feed Me integration](https://github.com/verbb/vizy/blob/craft-4/src/integrations/feedme/fields/Vizy.php) just doesn't know about that custom extension and so isn't preserving those id attributes. Is there a way to account for this in the plugin?

### Steps to reproduce

1. Add a Vizy extension, specifically one that is extending tiptap in our case
2. Import content through Feed Me that depends on that extension

### Craft CMS version

Craft Pro 4.8.0

### Plugin version

2.1.15

### Multi-site?

Yes

**Comment by engram-design** [comment]
Good point, we should probably be incorporating things into the Feed Me [editor](https://github.com/verbb/vizy/blob/999e13ee07d541348295022faf39f778c6593f32/src/integrations/feedme/fields/Vizy.php#L51-L66) instance.

Just wanted to confirm how you're registering your extensions though. Unless you're using `Nodes::EVENT_REGISTER_NODE` to register a custom node, the extension will only be a JS-class, which is used by the editor. We would need to have the same extension behaviour created and applied in PHP which would introduce a new event to register extensions.

**Comment by robmcfadden** [comment]
Got it, that makes sense. It is just a JS-class. I can resolve this for my situation outside of the Feed Me feed. It's a one-time feed and then content updates can resume in the entry itself where the Vizy/Tiptap extension works as expected. Thanks!
